src/js/main.js

```diff
@@ -23,7 +21,23 @@
     })//END OF RESULTS
 
 
-
   })//END OF MODULE
+  .controller('Find', ['$http', '$scope', function($http, $scope){
+    var BASEURL = 'http://wishcastr-staging.herokuapp.com/products/';
+
+    $scope.query = "";
+    $scope.products = { };
+    console.log($scope.query);
+    $scope.search = function(){
```

Comment:
A tracer bullet in here would help to know that you're actually triggering the function, too.